### ISSUE TITLE
Bump sbt-bintray to 0.5.1

### ISSUE THREAD

**Issue by chuwy** [snowplow/snowplow-scala-tracker#59]
Bintray exits with following error (not sure if sbt-bintray is able to fix it):

```
[info] Set current project to snowplow-scala-tracker (in build file:/home/travis/build/snowplow/snowplow-scala-tracker/)

java.lang.RuntimeException: failed to sync snowplow/snowplow-scala-tracker@0.4.0-M1 with maven central: {"status":"Sync Failed","messages":"[Failed to close repository: comsnowplowanalytics-1068. Server response:\n <nexus-error>\n  <errors>\n    <error>\n      <id>*<\u002fid>\n      <msg>Unhandled: Repository: comsnowplowanalytics-1068 has invalid state: closed<\u002fmsg>\n    <\u002ferror>\n  <\u002ferrors>\n<\u002fnexus-error>, Dropping existing partial staging repository.]"}
	at scala.sys.package$.error(package.scala:27)
	at bintray.BintrayRepo.syncMavenCentral(BintrayRepo.scala:139)
	at bintray.BintrayPlugin$$anonfun$bintrayPublishSettings$15.apply(BintrayPlugin.scala:112)
	at bintray.BintrayPlugin$$anonfun$bintrayPublishSettings$15.apply(BintrayPlugin.scala:110)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by chuwy** [comment]
Hey @alexanderdean, is it possible we're missing some permissions on JCenter/Maven Central that can result in above error?

**Comment by alexanderdean** [comment]
I don't think so.

**Comment by chuwy** [comment]
Problem was strangely resolved by bumping SBT to 1.0.2 (https://github.com/snowplow/snowplow-scala-tracker/issues/53)

**Comment by chuwy** [comment]
Unfortunately, it was not. Waiting for artifact to appear on Maven Central before pushing to master.
